bin/cmd.js

```diff
@@ -21,9 +21,9 @@ if (argv.h || argv.help) {
 var port = argv.port || argv.p
   , host = argv.host
 
-var server = hr(
-    { permament: argv.permament || argv['1']
-    }
-  )
+var server = hr({
+	permament: argv.permament || argv['1']
+	silent: argv.silent || argv.s
```

Comment:
> I don't get the reason. Is it because the object properties are supposed to be aligned properly?

The code style I'm using states that the comma should be at the start of the line and yes, also be aligned properly.

But even if you don't fix the code style, this is a syntax error now. Have you run the code?